commit 7d1311b93e58ed55f3a31cc8f94c4b8fe988a2b9 605f884d05cc0de8c3bde36281d58216011f51a5
Author:     Linus Torvalds <torvalds@linux-foundation.org>
AuthorDate: Sat Aug 16 10:40:26 2014 -0600
Commit:     Linus Torvalds <torvalds@linux-foundation.org>
CommitDate: Sat Aug 16 10:40:26 2014 -0600

    Linux 3.17-rc1

2	2	Makefile

diff --git a/Makefile b/Makefile
index 6aace67..e432442 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 VERSION = 3
-PATCHLEVEL = 16
+PATCHLEVEL = 17
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION = -rc1
 NAME = Shuffling Zombie Juror
 
 # *DOCUMENTATION*
