diff -au sawfish-1.3/config.h sawfish-1.3-patched/config.h
--- sawfish-1.3/config.h        Sun Aug 17 12:47:45 2003
+++ sawfish-1.3-patched/config.h        Sun Aug 17 12:54:06 2003
@@ -88,7 +88,7 @@
 #define HAVE_X11_EXTENSIONS_XINERAMA_H 1
 
 /* Define if <X11/Xft/Xft.h> exists */
-#define HAVE_X11_XFT_XFT_H 1
+/*#define HAVE_X11_XFT_XFT_H 1*/
 
 /* Define if you have <sys/wait.h> that is POSIX.1 compatible.  */
 #define HAVE_SYS_WAIT_H 1
diff -au -r sawfish-1.3/src/main.c sawfish-1.3-new/src/main.c
--- sawfish-1.3/src/main.c	2007-11-06 11:40:59.000000000 -0800
+++ sawfish-1.3-new/src/main.c	2003-01-12 12:35:23.000000000 -0800
@@ -50,7 +50,7 @@
 int exit_code = ec_no_exit;
 
 /* Saved value of argv[0] */
+static char *prog_name1;
-static char *prog_name;
 
 DEFSYM(sawfish_directory, "sawfish-directory");
 DEFSYM(sawfish_lisp_lib_directory, "sawfish-lisp-lib-directory");
@@ -346,9 +346,9 @@
     g_type_init();
 #endif
 
+    prog_name1 = *argv++; argc--;
-    prog_name = *argv++; argc--;
     lang = setlocale(LC_ALL, "");
+    rep_init (prog_name1, &argc, &argv, 0, 0);
-    rep_init (prog_name, &argc, &argv, 0, 0);
     stash_argv (old_argc, old_argv);
 
     if (rep_get_option ("--version", 0))
@@ -384,7 +384,7 @@
 \n\
     --version		print version details\n\
     --no-rc		don't load rc or site-init files\n\
+    --quit, -q		terminate the interpreter process\n", prog_name1);
-    --quit, -q		terminate the interpreter process\n", prog_name);
 	return 0;
    }
 
@@ -394,7 +394,7 @@
 
     rep_push_structure ("sawfish.wm");
 
+    if (sys_init(prog_name1))
-    if (sys_init(prog_name))
     {
 	sawfish_symbols();
 
